cli/azd/cmd/util.go

```diff
@@ -353,11 +351,20 @@ func openWithDefaultBrowser(ctx context.Context, console input.Console, url stri
 		return
 	}
 
-	fmt.Fprintf(
-		console.Handles().Stderr,
-		"warning: failed to open default browser: %s\n", err.Error(),
+	log.Printf(
+		"warning: failed to open default browser: %s\nTrying manual launch.", err.Error(),
 	)
 
+	// wsl manual launch
+	args := []string{
+		"-NoProfile", "-Command", fmt.Sprintf("Start-Process \"%s\"", url),
+	}
+	err = exec.Command("powershell.exe", args...).Run()
```

Comment:
Did we consider the [suggestion](https://github.com/Azure/azure-dev/issues/2291#issuecomment-1601326777) to use cmd instead? For me, the `az` powershell launch has a noticeable delay iirc.

Reply:
Sure, we can try `cmd` first.. and then fall back to `powershell` .  That would cover all.
I can also add a spinner while azd is trying to launch the browser.

Reply:
Updated to use commandRunner

Reply:
Updated to use cmd before powershell